spikingjelly/activation_based/neuron/flexsn.py

```
@@ -274,15 +274,86 @@ def __init__(
+                logging.warning(
+                    "FlexSN: could not import inductor kernel builders (%s); "
+                    "falling back to eager_scan/flex_sn_scan for all paths." % e
+                )
+                build_inference_kernel = None
+                build_training_kernels = None
+            if build_inference_kernel is not None:
+                try:
+                    self._inductor_scan_kernel, self._inductor_scan_info = (
+                        build_inference_kernel(core, num_inputs, num_states, num_outputs)
+                    )
+                except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)